File: app/src/main/java/com/reed/reedplayer/activity/VideoPlayerActivity.java
package com.reed.reedplayer.activity;

import com.reed.reedplayer.fragment.BaseFragment;
import com.reed.reedplayer.fragment.VideoPlayerFragment;

/**
 * Created by thinkreed on 16/7/22.
 */
public class VideoPlayerActivity extends FullScreenActivity {
  @Override
  protected BaseFragment getFragment() {
    return VideoPlayerFragment.newInstance(getIntent().getExtras());
  }
}

File: app/src/main/java/com/reed/reedplayer/activity/VideoPlayerActivity.kt
package com.reed.reedplayer.activity

import com.reed.reedplayer.fragment.BaseFragment
import com.reed.reedplayer.fragment.VideoPlayerFragment

/**
 * Created by thinkreed on 16/7/22.
 */
class VideoPlayerActivity : FullScreenActivity() {
    override fun getFragment(): BaseFragment {
        return VideoPlayerFragment.newInstance(intent.extras)
    }
}
